website/docs/components/accordion/partials/code/how-to-use.md

```diff
@@ -198,6 +198,12 @@ The `ariaLabel` value is applied to the HTML button which controls visibility of
 
 ### Title tag
 
+!!! Insight
+
+The default `@titleTag` is `"div"` because the correct value is dependent on the individual page. We strongly encourage consumers to update the `@titleTag` to meet WCAG Success Criterion [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html) as the visual experience should match what is presented to the user with assistive technology.
+
```

Comment:
See suggestions I added for a similar banner:
https://github.com/hashicorp/design-system/pull/2897/files#r2114906824